README.md

```diff
@@ -60,4 +60,7 @@ $ docker run --env-file .env.test  -it  mytest
 
 #prod 
 $ docker build --target production --tag prod .
-$ docker run --env-file .env -it -p 5000:80  prod
\ No newline at end of file
+$ docker run --env-file .env -it -p 5000:80  prod
+
+# Link to app that is hosted
+http://devopscourse1509.herokuapp.com/
```

Comment:
I think this changed (and also, adding `<>` around the outside should tell the Markdown to make it a clickable link)
```suggestion
<http://testingdevopscourse1509.herokuapp.com/>
```